templates/theories/linear_algebra.md

```diff
@@ -2,105 +2,110 @@
 (which incidentally allows actions such as `matrix.mul_vec`),
 while linear maps are in their essence an action on vectors
 (which incidentally can be represented by a matrix if we have a finite basis).
 If you want to do computations, a matrix is a better choice.
 If you want to do proofs without computations, a linear map is a better choice.
 
-The type `general_linear_group R M` is the group of invertible `R`-linear maps from `M` to itself.
+The type [`general_linear_group R M`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/basic.html#linear_map.general_linear_group) is the group of invertible `R`-linear maps from `M` to itself.
 `general_linear_equiv R M` is the equivalence between `general_linear_group` and `M ≃[R]ₗ M`.
 `special_linear_group.to_GL` is the embedding from the special linear group (of matrices) to the general linear group (of linear maps).
 
-The file `linear_algebra.matrix` connects matrices with linear maps.
-`linear_equiv_matrix'` shows that `matrix.mul_vec` is a linear equivalence between `matrix m n R` and `(n → R) →[R]ₗ (m → R)`.
-In addition, `linear_equiv_matrix` takes a basis `ι` for `M₁` and `κ` for `M₂`
-and gives the equivalence between `R`-linear maps between `M₁` and `M₂` and `matrix ι κ R`.
-If you have an explicit basis for your maps, this equivalence allows you to do calculations such as getting the determinant.
-
-The dual space, consisting of linear maps `M →[R]ₗ R`, is defined in `linear_algebra.dual`.
+The dual space, consisting of linear maps `M →[R]ₗ R`, is defined in [`linear_algebra.dual`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/dual.html).
 
 ### Bilinear, Sesquilinear and Quadratic Forms ###
 
-#### `linear_algebra.bilinear_form` ####
+#### [`linear_algebra.bilinear_form`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/bilinear_form.html) ####
 
 For an `R`-module `M`, the type `bilin_form R M` is the type of maps `M → M → R` that are linear in both arguments.
 The equivalence between `bilin_form R M` and maps `M →ₗ[R] M →ₗ[R] R` that are linear in both arguments is called `bilin_linear_map_equiv`.
 A matrix `M` corresponds to a bilinear form that maps vectors `v` and `w` to `row v ⬝ M ⬝ col w`.
-The equivalence between `bilin_form R (n → R)` and `matrix n n R` is called `bilin_form_equiv_matrix`.
+The equivalence between `bilin_form R (n → R)` and `matrix n n R` is called [`bilin_form_equiv_matrix`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/bilinear_form.html#bilin_form_equiv_matrix).
 
-#### `linear_algebra.sesquilinear_form` ####
+#### [`linear_algebra.sesquilinear_form`](https://leanprover-community.github.io/mathlib_docs/linear_algebra/sesquilinear_form.html) ####
 
 For an `R`-module `M`, the type `sesq_form R M` is the type of maps `M → M → R` that are linear in the first argument and antilinear in the second.
```

Comment:
You are still missing the involution here.

Reply:
You are correct! I believe I got it right, now.

Reply:
Thanks!